### ISSUE TITLE
Wd Mycloud Gen 2

### ISSUE THREAD

**Issue by machsix** [WDCommunity/wdpksrc#45]
Hi! I'm maintaining some packages for WD MyCloud Gen2.
My repo is https://github.com/machsix/WDMyCloud-Gen2
Is it possible to join the community?

**Comment by stefaang** [comment]
@machsix I've added your repo to the wiki.
You're more than welcome to add a few of your packages here, but I won't accept binary blobs without source in this repo.

**Comment by machsix** [comment]
> @machsix I've added your repo to the wiki.
> You're more than welcome to add a few of your packages here, but I won't accept binary blobs without source in this repo.

Thank you! I have restructured the repository. Basically, one folder is for Debian on WD, and the other one is for OS3. I agree that license info should be respected and provided and these kinds of contents will be detailed in the future. I also think OS3 is kind of dead although WD officially still updates the firmware. The obsolete kernel and the way sysroot handled by the device increase the difficulty to migrate software. I feel Debian or Archlinux ARM is the way to go, although fox has stopped working on it for WD Mycloud Gen2. They are more reliable (maintained by the community) and have better compatibility (We don't need to reinvent wheel for different device). I have compiled the latest kernel and some "optimized" software for WD MyCloud Gen2.
